app/views/dashboards/_subject.html.erb

```diff
@@ -23,6 +23,19 @@
   </div>
 <% end %>
 
+<% if user.homepage_url.present? %>
+  <div class="flex items-center mb-4 text-b3 lg:text-b2">
+    <%= icon_tag("link", color: :primary, class: "w-6 text-orange mr-3") %>
+    <p class="text-neutral-800 dark:text-white"><%=
+      link_to(
+        user.homepage_url,
+        user.homepage_url,
+        rel: "nofollow"
```

Comment:
opps sorry I didn't realize there was a final decision on the `rel="nofollow"`.  Thank you for adding these changes :) @martinemde